README.md

```diff
@@ -76,6 +76,10 @@ Also, Kruize HPO can be installed natively on Linux, as a container or in miniku
 You can run a specific version of the Kruize HPO container
     `$ ./deploy_hpo.sh -c minikube -o image:tag`
 
+## Kruize HPO on Operate First
+We have deployed Kruize HPO on Operate first community cloud using namespace 'openshift-tuning', to promote open operations.You can access HPO on operate first by running the following command:
+$ ./deploy_hpo.sh -c operate-first
+
```

Comment:
Since users might not be aware of OperateFirst, please add an one liner about OperateFirst and also provide a link to it